translations/translations.json

```
@@ -268,12 +268,14 @@
   },
   "chat": {
     "de": {
+      "botIcon": "Chatbot",
       "button": "Chat (beta)",
       "conversationTitle": "Bitte gib deine Frage in das Textfeld ein.",
       "conversationText": "Du kannst alles fragen, von lokalen Informationen bis hin zu spezifischen Anfragen bezüglich deiner Situation.",
       "dataSecurity": "Datenschutz und Sicherheit: Deine Privatsphäre ist uns sehr wichtig. Alle Gespräche werden vertraulich behandelt und deine Daten werden sicher verarbeitet. Mehr Informationen zum Datenschutz findest du in unseren Datenschutzrichtlinien.",
       "errorMessage": "Es ist ein Fehler bei der Kommunikation aufgetreten.",
       "header": "{{appName}} Chat Support",
+      "humanIcon": "Berater*in",
```

:wrench: See comment above
```suggestion
      "human": "Berater*in",
```